### ISSUE TITLE
Doesn't work in production on Rails 5

### ISSUE THREAD

**Issue by paul-at** [themilkman/jquery-tablesorter-rails#7]
I'm having a strange problem with table sorter on Rails 5 when in production:
` $("#applicationList").tablesorter is not a function. `

The application is fairly basic, there is no complex javascript that might interfere, just stock jQuery+jquery_ujs and turbolinks plus bootstrap.

The problem is reproducible in development mode as well, by setting  config.assets.debug = false

I've manually checked the order of javascript in concatenated file and it is the same as order of individual` <script>` tags in debug mode.

I feel truly stuck at this point, would appreciate any ideas regarding what may be wrong.

**Comment by themilkman** [comment]
Hi @paul-at, 
currently I am working on a rails 5 app with tablesorter (couldn't test it in a production env right now), but for me it works fine (I did also try the assets.debug flag). Could this be related to turbolinks (there were a few big changes in the latest version)?

In my coffeescript code I use:

```coffee
document.addEventListener 'turbolinks:load', ->
  $('table.tablesorter').each ->
    $(this).tablesorter( ...
```

**Comment by paul-at** [comment]
Thank you for your help! By trying to create a test case using your input I managed to detect the problem. Its been caused by https://github.com/twbs/bootstrap-rubygem

Turns out Bootstrap tooltips and popovers included by default depend on a third-party JS lib which is not included by default. I assumed the related error was only a warning, but turned out it prevented JS from loading properly when concatenated.
